### ISSUE TITLE
2018.11.14 wednesday talk-by Jason C and XiaoGang lv

### ISSUE THREAD

**Issue by CherishingYC** [CherishingYC/blog#2]
Jason C

- Q:对于一个给定的网络（不可更改），你有多少种方法去将输出精度提高（做优化的方法）？
A：思路：抛去CNN，只剩下输入输出
      right answer: data  Augment（基于性质：平移不变性）

- detail：
①Keras有个毛病：只有输出，没有输入
②Tensorboard：一个工具包，可视化学习
③新浪微博：爱可可（推荐科技文章），（Wechat）机器之心       

- propose：
①replace code rcnn（Keras） with code faster rcnn（Keras）
                                              or mask （Keras / pytorch）
②about 11th weekly conclusion：
fast rcnn将分类和回归都放到了CNN网络中
my question：Why？ Advantages？How？
C's answer：首先，可以这么做，第二这么做的优点就是不同的任务可以促进，或者优化基础特征提取的过程。至于如何实现，fast以后所有加RPN结构，做检测的，都是分类加回归，没必要看了，因为太老了。

XiaoGang lv
- Q：关于minist数据集你了解多少？
  A：none。
此处建议，了解框架，深入了解代码细节，目标：深入掌握分类
             要求：实现将minist数据集换成ciffar10的
                        实现将Keras框架改成pytorch框架

- detail：training：训练集
               validation：验证集
               test：测试集
training与validation是一起送入网络训练的，不是分开的
test是没标签的

- propose：继续学习faster-rcnn
                  参照Keras文档，熟悉代码
